less/index.less

```diff
@@ -57,3 +57,175 @@ html, body {
     height: 100%;
 }
 
+@primary: #203731;
```

Comment:
I see you have these two variables and you know how to correctly call/use them in the project but I'm not seeing the mixin anywhere. make sure you make one and use it.